kv-components/vue/KvButton.vue

```diff
@@ -30,44 +46,60 @@
 <script>
 export default {
 	props: {
+		/**
+		 * Use if linking to a Vue route
+		 * */
 		to: {
 			type: String,
 			default: null,
 		},
+		/**
+		 * Use if linking to an external link or old-stack page
+		 * */
 		href: {
 			type: String,
 			default: null,
 		},
-		disabled: {
-			type: Boolean,
-			default: false,
+		/**
+		 * The behavior of the button when used in an HTML form.
+		 * `button (default), submit, reset`
+		 * */
+		type: {
+			type: String,
+			default: 'button',
+			validator(value) {
+				return ['button', 'submit', 'reset'].includes(value);
+			},
 		},
 		/**
 		 * Appearance of the button
-		 * `primary (default), secondary, link, danger`
+		 * `primary (default), secondary, danger, link, ghost`
```

Comment:
I don't love the name 'link' for this button appearance.  Any other ideas?
![image](https://user-images.githubusercontent.com/187937/117369422-be2f9f80-ae79-11eb-8751-b7c566b04c01.png)

Reply:
Kinda seems like Ghost "is" the new version of Text link...

Otherwise I think the discussion around a text link really comes down to the type style and the padding of the element...